### PR TITLE
Feature 664

### PR DESCRIPTION
**Pull request by sujata-m**
- Updated Request and Response Payload 
- Fixed Unit, Integration and Harness according to new request and response payload.

Request Payload - 
```
{
  "messageId": "c8c76e89f30944d2b2abd2491bd95337",
  "messageType": "workflow_identifier",
  "data": {
    "file_upload_path": "https://tdeisamplestorage.blob.core.windows.net/osw/test_upload/osw.zip",
    "tdei_project_group_id": "0b41ebc5-350c-42d3-90af-3af4ad3628fb"
  }
}
```

Response Payload - 
```
{
  "messageId": "c8c76e89f30944d2b2abd2491bd95337",
  "messageType": "workflow_identifier",
  "data": {
    "file_upload_path": "https://tdeisamplestorage.blob.core.windows.net/osw/test_upload/osw.zip",
    "tdei_project_group_id": "0b41ebc5-350c-42d3-90af-3af4ad3628fb",
    "source_url": "https://tdeisamplestorage.blob.core.windows.net/osw/test_upload/osw.zip",
    "formatted_url": "https://tdeisamplestorage.blob.core.windows.net/osw/test_upload/osw.zip",
    "success": true,
    "message": ""
  }
}
```

On Demand Request Payload - 
```
{
  "messageId": "c8c76e89f30944d2b2abd2491bd95337",
  "messageType": "workflow_identifier ON_DEMAND",
  "data": {
    "sourceUrl": "https://tdeisamplestorage.blob.core.windows.net/osw/2023/11/c552d5d1-0719-4647-b86d-6ae9b25327b7/aff14a0d29ab4acbaef639063462e85b/naresh-som-2.zip",
    "jobId": "42",
    "source": "osw",
    "target": "osm"
  }
}
```

On Demand Response Payload - 
```
{
  "messageId": "c8c76e89f30944d2b2abd2491bd95337",
  "messageType": "workflow_identifier ON_DEMAND",
  "data": {
    "sourceUrl": "https://tdeisamplestorage.blob.core.windows.net/osw/2023/11/c552d5d1-0719-4647-b86d-6ae9b25327b7/aff14a0d29ab4acbaef639063462e85b/naresh-som-2.zip",
    "jobId": "42",
    "source": "osw",
    "target": "osm",
    "formattedUrl": "https://tdeisamplestorage.blob.core.windows.net/osw/2023/11/c552d5d1-0719-4647-b86d-6ae9b25327b7/aff14a0d29ab4acbaef639063462e85b/naresh-som-2.zip",
    "success": true,
    "message": ""
  }
}
```

### CODE REVIEW

**Review by susrisha — APPROVED** [review]

File: src/models/osw_validation_message.py
Comment (sureshgaussian) [inline src/models/osw_validation_message.py:1]: why do we have osw_validation_message.py in OSW formatter service? Is this a wrong name for the formatter message?

**Review by sureshgaussian — CHANGES_REQUESTED** [review]
I couldn't figure out the env variables needed to run the unit and integration tests. I also do not know why we have OSWValidationMessage model in formatter. If you can attach output of the integration tests to this PR I will approve.

File: src/models/osw_validation_message.py
Comment (susrisha) [reply src/models/osw_validation_message.py:1]: The incoming message for osw-formatter is got through `osw-validation` topic (re-using for regular flow and on-demand flow). I believe thats  why we have the osw_validation_message.py

Comment (sureshgaussian) [reply src/models/osw_validation_message.py:1]: formatter service wont know if the service before it is a regular flow, or on demand flow. It should just get the input file location, and where to write the formatted output. I will approve the change for now as this is not a huge issue. However, I would like to have a code review in Jan to clean up unnecessary things.

Comment (susrisha) [reply src/models/osw_validation_message.py:1]: Understood. We will relook at the services after this.

**Pull request merged**